### PR TITLE
chore: Add changelog for release v49.12.0

### PR DESCRIPTION
**Pull request by AnanyaJha**
### What does this PR do?
This PR adds the changelog for release v49.12.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Show progress bar when installing dependencies for Function Create ([PR #2563](https://github.com/forcedotcom/salesforcedx-vscode/pull/2563))
+
+- Add feature flag for Functions ([PR #2572](https://github.com/forcedotcom/salesforcedx-vscode/pull/2572))
+
+- Show message if no default org is set when starting Function ([PR #2560](https://github.com/forcedotcom/salesforcedx-vscode/pull/2560))
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md:11]: @jag-sfdc @ekapner are we adding Functions related things to docs yet? If yes, we can build out these entries more

Comment (ekapner) [reply packages/salesforcedx-vscode/CHANGELOG.md:11]: FYI @danyusf 

Per @jag-sfdc's doc, you need to go into Settings to enable the extension - but unsure if we want to announce its existence to the entire world.  Functions is having a closed pilot on Oct 19 - so I'm leaning toward that we skip documenting anything here - but let Dan document how to enable the extension in his pilot docs.

Comment (ekapner) [reply packages/salesforcedx-vscode/CHANGELOG.md:11]: Once Functions goes to public beta, we can make an announcement about the extension.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+
+#### salesforcedx-vscode-core
+
+- Manage org context through WorkspaceContext object ([PR #2549](https://github.com/forcedotcom/salesforcedx-vscode/pull/2549))
+
+#### salesforcedx-vscode-soql
+
+- We fixed some minor under-the-hood bugs.
+
+- Lock soql-builder-ui and soql-language-server versons ([PR #2568](https://github.com/forcedotcom/salesforcedx-vscode/pull/2568))
```
Comment (ekapner) [inline packages/salesforcedx-vscode/CHANGELOG.md:27]: typo: versions

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+
+- Show progress bar when installing dependencies for Function Create ([PR #2563](https://github.com/forcedotcom/salesforcedx-vscode/pull/2563))
+
+- Add feature flag for Functions ([PR #2572](https://github.com/forcedotcom/salesforcedx-vscode/pull/2572))
+
+- Show message if no default org is set when starting Function ([PR #2560](https://github.com/forcedotcom/salesforcedx-vscode/pull/2560))
+
+#### salesforcedx-vscode-soql
+
+- Add textmate grammar for SOQL files ([PR #2562](https://github.com/forcedotcom/salesforcedx-vscode/pull/2562))
```
Comment (ekapner) [inline packages/salesforcedx-vscode/CHANGELOG.md:15]: TextMate grammar

Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md:15]: Just like with functions, we need to remove everything related to vscode-soql until it gets published

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,31 @@
+# 49.12.0 - September 30, 2020
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Show progress bar when installing dependencies for Function Create ([PR #2563](https://github.com/forcedotcom/salesforcedx-vscode/pull/2563))
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]: I'm not sure we're going to want to include anything related for Functions yet.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,31 @@
...
+
+- Add feature flag for Functions ([PR #2572](https://github.com/forcedotcom/salesforcedx-vscode/pull/2572))
+
+- Show message if no default org is set when starting Function ([PR #2560](https://github.com/forcedotcom/salesforcedx-vscode/pull/2560))
+
+#### salesforcedx-vscode-soql
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md:13]: Same with soql - I don't believe we'll want to include anything in the changelog for these features yet

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 1 above)
Comment (AnanyaJha) [reply packages/salesforcedx-vscode/CHANGELOG.md:11]: Sounds good, I'll remove these entries for now

**Review by sfsholden — APPROVED** [review]

**Review by ekapner — APPROVED** [review]

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 1 above)
Comment (jag-sfdc) [reply packages/salesforcedx-vscode/CHANGELOG.md]: Sorry missed this conversation. Yes, we shouldn't make an announcement yet.

**Pull request merged**